src/util.cpp

```diff
@@ -97,15 +97,15 @@ namespace boost {
 
 } // namespace boost
 
-using namespace std;
+// blank space for binary match
```

Comment:
The binary check tool trips over the LOCK macros. If a line number is removed, the binary check ends up failing.